pkgs/native_toolchain_c/lib/native_toolchain_c.dart

```diff
@@ -5,5 +5,8 @@
 /// A library to invoke the native C compiler installed on the host machine.
 library;
 
-export 'src/cbuilder/cbuilder.dart';
+export 'src/cbuilder/cbuilder.dart' show CBuilder;
```

Comment:
Not exposing CTool here is the right thing `dart doc` wise I think, e.g. the fields show up in both `CBuilder` and `CLinker` without showing the base class. Do you agree?